File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/example/Breed.java
package com.raizlabs.android.dbflow.test.example;

import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
 * Description:
 */
@Table(database = TestDatabase.class)
public class Breed extends BaseModel {

    @PrimaryKey
    long breed_id;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/example/Breed.kt
package com.raizlabs.android.dbflow.test.example

import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel
import com.raizlabs.android.dbflow.test.TestDatabase

/**
 * Description:
 */
@Table(database = TestDatabase::class)
class Breed : BaseModel() {

    @PrimaryKey
    var breed_id: Long = 0
}
